src/controllers/admin.js

```diff
@@ -6,10 +6,11 @@ const createAdmin = async (req, res) => {
     firstName, lastName, dni, phone, email, city, password,
   } = req.body;
   try {
-    const found = await Admin.findOne({ dni });
-    if (found) {
-      return res.status(404).json({
-        message: `An admin with DNI ${dni} already exists`,
+    const dniExists = await Admin.findOne({ dni });
+    const emailExists = await Admin.findOne({ email });
```

Comment:
You could find both values in just one line like this:
`const exists = await Admin.findOne({ $or: [{ dni }, { email }]`